.gitignore

```diff
@@ -1,2 +1 @@
-Cargo.lock
```

Comment:
This is a library crate, so `Cargo.lock` has no effect on dependent crates. It is correct to ignore this file.